config/common_versions.yml

```diff
@@ -0,0 +1,5 @@
+title: master
```

Comment:
Is master correct for this branch?

Reply:
Probably not... I updated it, I think that's correct?